lib/routes/account.js

```diff
@@ -1477,6 +1478,8 @@ module.exports = (
               session.lastAccessTimeFormatted = localizeTimestamp.format(session.lastAccessTime,
                 request.headers['accept-language'])
 
+              session.os = session.uaOS || 'Unknown'
```

Comment:
Are you returning this to be displayed in the UI? Wouldn't we want `'Unknown'` to be localised in that case?

Just wondering whether it's better to leave the auth server to return `null` or the empty string or whatever, then have the content server responsible for the default string...?